setup.cfg

```diff
@@ -0,0 +1,28 @@
+[metadata]
+name = json-schema-for-humans
+description = Generate static HTML documentation from JSON schemas
+description-file = README.md
+author = Denis Blanchette, AbdulRahman Al Hamali
+author-email = tools@coveo.com
+license = Apache Software License
+url = https://github.com/coveooss/json-schema-for-humans
+classifiers =
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.7
+    License :: OSI Approved :: Apache Software License
```

Comment:
V2

Reply:
I thought so too, but I am only using those that are listed here: https://pypi.org/classifiers/